### DISCUSSION TITLE
Gluon trainer on GPU with CUDA 11

### DISCUSSION THREAD

**Discussion by nils-fl** [awslabs/gluonts#1210] [Q&A] [answered]
Hi and thanks for this awesome library. 

I am trying to train a model on GPU using the nightly build mxnet-cu110 (2.0.0b20201208-py3 from https://dist.mxnet.io/python). 
It throws:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-6-4afada510220> in <module>
----> 1 from gluonts.model.simple_feedforward import SimpleFeedForwardEstimator
      2 from gluonts.model.n_beats import NBEATSEnsembleEstimator
      3 from gluonts.mx.trainer import Trainer

~/miniconda3/envs/autopilot/lib/python3.8/site-packages/gluonts/model/simple_feedforward/__init__.py in <module>
     13 
     14 # Relative imports
---> 15 from ._estimator import SimpleFeedForwardEstimator
     16 
     17 __all__ = ["SimpleFeedForwardEstimator"]

~/miniconda3/envs/autopilot/lib/python3.8/site-packages/gluonts/model/simple_feedforward/_estimator.py in <module>
     19 
     20 # First-party imports
---> 21 from gluonts.core.component import validated
     22 from gluonts.dataset.field_names import FieldName
     23 from gluonts.model.estimator import GluonEstimator

~/miniconda3/envs/autopilot/lib/python3.8/site-packages/gluonts/core/component.py in <module>
    223 
    224 
--> 225 @equals.register(mx.gluon.ParameterDict)
    226 def equals_parameter_dict(
    227     this: mx.gluon.ParameterDict, that: mx.gluon.ParameterDict

AttributeError: module 'mxnet.gluon' has no attribute 'ParameterDict'
```
Am I at the right place to ask if there is a timeline on when gluon is usable with CUDA 11?

Cheers
Nils

**Comment by lostella** [comment, marked as answer]
@nils-fl I moved this to “discussions” (we just started using this new GitHub feature), hope you don’t mind.

I think the issue is not caused by cu11 but rather mxnet 2.0. We currently test on mxnet<2 only, and from your stack trace there seem to be some breaking changes. We haven’t really started looking into mxnet 2.0, we will make a plan for this as soon as an ETA for it is more concrete.

If you’re interested in cu11 specifically, have you checked if mxnet 1.8 will support it?

**Comment by nils-fl** [comment]
Thanks Lorenzo for moving this to discussions, makes more sense. I wasn't aware of it.

You are absolutely right! I somehow missed that there is a mxnet 1.9 cu11 nightly available. It works perfectly.

Thanks!
